### PR TITLE
Update Target and Min SDK version

### PR DESCRIPTION
**Pull request by LanceMcCarthy**
Needed a tweak so that the project will load with the latest SDK (16232).

### CODE REVIEW

File: Demos/MaterialCreator/MaterialCreator.csproj [hunk 1]
```diff
@@ -11,8 +11,8 @@
     <AssemblyName>MaterialCreator</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.16225.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.16225.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.16257.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.16257.0</TargetPlatformMinVersion>
```
Comment (daneuber) [inline Demos/MaterialCreator/MaterialCreator.csproj]: Please change these back to 16232 so we can accept; 16257 is not available yet

Comment (LanceMcCarthy) [inline Demos/MaterialCreator/MaterialCreator.csproj]: It just got released an hour ago, but let me double check that it loads. I'll sort back asap with results.

Comment (LanceMcCarthy) [inline Demos/MaterialCreator/MaterialCreator.csproj]: Confirmed, double checked and just ran it again using today's [public Insider Preview SDK release](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK?wa=wsignin1.0) (16257). 

Thanks Danielle!

**Comment by daneuber** [comment]
Merged. Thanks for contributing!

**Pull request merged**